DOJ-8: Adding universal form fields.

Tom-Camp opened the pull request.

malikkotob commented on docroot/modules/custom/webform_template/config/install/webform_template.settings.yml:
```diff
@@ -1 +1 @@
-webform_template_elements: "first_name:\n  '#type': textfield\n  '#title': 'First Name'\n  '#required': true\nlast_name:\n  '#type': textfield\n  '#title': 'Last name'\n  '#required': true\nemail:\n  '#type': email\n  '#title': Email\n  '#required': true\nrequest_description:\n  '#type': textarea\n  '#title': 'Describe the information you''re requesting'\n  '#required': true\n"
+webform_template_elements: "prefix_title:\n  '#type': select\n  '#title': Prefix/Title\n  '#options': titles\nfirst_name:\n  '#type': textfield\n  '#title': 'First Name'\n  '#required': true\nmiddle_initial_middle_name:\n  '#type': textfield\n  '#title': 'Middle Initial/Middle Name'\nlast_name:\n  '#type': textfield\n  '#title': 'Last Name'\n  '#required': true\nsuffix:\n  '#type': select\n  '#ti ...
```
A few items:
1. I noticed the state dropdown isn't dependent on the selected country. Would it be a large lift to get that functionality?
1. Typo: `non comercial`: "comercial" should be "commercial"
1. Looking at the excel spreadsheet, it seems like the expectation for the `Request Fee Waver` and `Request Expedited Processing` fields should have text fields appear if 'Yes' is selected to allow the user to enter an explanation.
1. Mind if we remove the "I agree to the privacy policy" checkbox?
1. Mind if we remove the text for the Email field that says "We collect this information in accordance with Regulation 1.3.5s7 and here's showing how a link works."

Tom-Camp commented:
@malikkotob - 
1) I conditionalized the State dropdown to only show when the country is United States
2) Fixed the typo
3) Added conditional text area fields for explanations for the Request Fee Waiver and Request Expedited Processing fields which only show when the parent field is set to Yes
4) Removed the Policy checkbox and the Email description.

Tom-Camp commented:
I also updated the _Functional Review_ steps to create a form and to test the conditional fields.
